### PR TITLE
Replace Admonition Style

### PR DESCRIPTION
**Pull request by milldr**
## what
- Replace all Docusarus style admonitions with GitHub style admonitions

## why
- We now can provide the GH style here and have the docs site convert the format into Docusarus style

## references
- DEV-2453
- Required by https://github.com/cloudposse/docs/pull/642

### CODE REVIEW

File: modules/eks/cluster/CHANGELOG.md [hunk 1]
```diff
@@ -65,18 +65,18 @@ In this release, this component only supports assigning "team roles" to Kubernet
...
-access to the EKS control plane.
-
-This may seem like it makes it easier to secure the EKS control plane, but Terraform users will still require network
-access to the EKS control plane to manage any deployments or other Kubernetes resources in the cluster. This means that
-this upgrade does not substantially change the need for network access.
-
-:::
+> [!TIP]
+>
+> Network Access Considerations
```
Comment (milldr) [inline modules/eks/cluster/CHANGELOG.md:70]: GitHub admonitions do not support titles 😦

File: modules/auth0/tenant/README.md [hunk 2]
```diff
@@ -42,11 +42,11 @@ in Terraform. Follow the
...
 create a Machine to Machine application.
 
-:::tip Machine to Machine App Name
-
-Use the Context Label format for the machine name for consistency. For example, `acme-plat-gbl-prod-auth0-provider`.
-
-:::
+> [!TIP]
+>
+> Machine to Machine App Name
```
Comment (osterman) [inline modules/auth0/tenant/README.md:47]:
```suggestion
> #### Machine to Machine App Name
```

File: deprecated/eks/karpenter-provisioner/README.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-
-With v1beta1 of Karpenter, the `provisioner` component is deprecated.
-Please use the `eks/karpenter-node-group` component instead.
-
-For more details, see the [Karpenter v1beta1 release notes](/modules/eks/karpenter/CHANGELOG.md).
-
-:::
+> [!WARNING]
+>
+> This component is DEPRECATED
```
Comment (osterman) [inline deprecated/eks/karpenter-provisioner/README.md:5]:
```suggestion
> #### This component is DEPRECATED
```

File: modules/aws-config/README.md [hunk 4]
```diff
@@ -20,25 +20,25 @@ Some of the key features of AWS Config include:
...
-    make sure to store them somewhere else (i.e. S3) so the findings are not lost.
-  - See the
-    [Audit Manager docs](https://aws.amazon.com/blogs/mt/integrate-across-the-three-lines-model-part-2-transform-aws-config-conformance-packs-into-aws-audit-manager-assessments/)
-    if you think you would like to convert conformance packs to custom Audit Manager assessments.
-- The maximum number of AWS Config conformance packs that can be created in a single account is 50.
-
-:::
+> [!WARNING]
+>
+> AWS Config Limitations
```
Comment (osterman) [inline modules/aws-config/README.md:25]:
```suggestion
> #### AWS Config Limitations
```

File: modules/auth0/tenant/README.md (hunk 2 above)
Comment (milldr) [reply modules/auth0/tenant/README.md]: Without converting anything docusaurus recognized the title with `####`!

This is what's in the mdx formatted for docusaurus:
```
:::tip

#### Machine to Machine App Name

Use the Context Label format for the machine name for consistency. For example, `acme-plat-gbl-prod-auth0-provider`.

:::
```

And this is the result:
CleanShot 2024-08-07 at 12 42 24@2x

Comment (milldr) [reply modules/auth0/tenant/README.md]: and here's the GH style:
CleanShot 2024-08-07 at 12 45 31@2x

**Review by osterman — APPROVED** [review]

File: modules/eks/actions-runner-controller/CHANGELOG.md [hunk 5]
```diff
@@ -76,12 +76,12 @@ of memory allocated to the runner Pod to account for this. This is generally not
...
 
-:::warning You must configure a memory request for the runner Pod
-
-When using `tmpfs_enabled`, you must configure a memory request for the runner Pod. If you do not, a single Pod would be
-allowed to consume half the Node's memory just for its disk storage.
-
-:::
+> [!WARNING]
+>
+> You must configure a memory request for the runner Pod
```
Comment (osterman) [inline modules/eks/actions-runner-controller/CHANGELOG.md:81]:
```suggestion
> #### You must configure a memory request for the runner Pod
```

**Review by osterman — APPROVED** [review]

**Pull request merged**